### DISCUSSION TITLE
Is it possible to use devtools for a Chrome extension

### DISCUSSION THREAD

**Discussion by hwillson** [apollographql/apollo-client-devtools#382] [Ideas]
Migrated from: https://github.com/apollographql/apollo-client-devtools/issues/144

I can't get the Apollo pane to show up in the popup.html devtools or background page devtools. For react devtools, they provide a standalone Electron app (https://github.com/facebook/react-devtools/tree/master/packages/react-devtools) that lets you inspect the Chrome extension's React tree.

**Comment by jerelmiller** [comment]
We now create an Apollo panel regardless of whether an Apollo Client instance is found or not to avoid issues where the detection might be flaky.

For the standalone build, this is captured in https://github.com/apollographql/apollo-client-devtools/discussions/1046 so I'm going to go ahead and close this as resolved 🙂
